pkg/ast/pipesearch/searchHandler.go

```diff
@@ -459,6 +459,10 @@ func convertRRCsToJSONResponse(rrcs []*sutils.RecordResultContainer, sizeLimit u
 	}
 
 	allJsons, allCols, err := record.GetJsonFromAllRrc(rrcs, false, qid, segencmap, aggs)
+	if aggs != nil && aggs.TransactionArguments != nil && err == nil {
```

Comment:
the below err != nil check shoule be before this if block: a practice we follow, always process the err first and then continue with the rest of the processing

Reply:
Yes got it! I will change it!